react/src/utils/theme.ts

```diff
@@ -1,23 +1,50 @@
 import { extendTheme } from "@chakra-ui/react";
 
 const colors = {
-  brand: {
-    900: "#1a365d",
-    800: "#153e75",
-    700: "#2a69ac",
+  brandRed: {
+    300: "#ef404a",
+    200: "#f37167",
+    100: "#f8aa9e",
   },
-  primary: {
-    700: "light-blue",
-    500: "blue",
+  brandBlue: {
+    300: "#29335f",
+    200: "#315c88",
+    100: "#aacfe3",
   },
+  brandYellow: {
+    300: "#d89016",
+    200: "#e4aa4f",
+    100: "#f4e6a0",
+  },
+  brandGray: "#848689",
+  // this was additional generated on https://coolors.co/ to have a fitting green
+  brandGreen: "#60a561"
 };
 
 export const theme = extendTheme({
   colors,
   components: {
     Button: {
-      defaultProps: {
-        borderRadius: "0",
+      baseStyle: {
+        borderRadius: 0,
+      },
+      // TODO: we need to define our brand color palette and apply a whole colorScheme to config chakra UI properly.
```

Comment:
Can you turn this into a trello card and remove the todo here :sunglasses: maybe link the file or the line in the card

Reply:
I linked the line in the trello card 
https://trello.com/c/wD3TuF2T

Reply:
I disagree with removing the TODO here